graphqlapi/src/main/java/graphql/kickstart/servlet/CustomGraphQLServlet.java

```diff
@@ -86,11 +92,14 @@ private void handle(HttpServletRequest request, HttpServletResponse response) {
       if (requestHandler == null) {
         failOnUnknownKeyspace(keyspaceName, response);
       } else {
+        Timer.Context timerContext = metricRegistry.timer(keyspaceName).time();
```

Comment:
For basic metrics maybe we should leverage something like?

https://metrics.dropwizard.io/4.1.2/manual/servlet.html

https://github.com/dropwizard/metrics/tree/release/4.1.x/metrics-servlet/src/main/java/com/codahale/metrics/servlet 

It looks like it keeps track of error codes which would be important for determining service health? Also, it would be nice to leverage something common for all http based services without having to write something custom for each.

Reply:
Most of them we can get away with annotations.

Reply:
💯  This PR is a short-term fix, I'm thinking of removing the custom servlets and migrating to Dropwizard for uniformity with other modules.

But in the meantime, I agree that the metrics servlet is a better solution, I'll make the change.